build-system/tasks/runtime-test/status-report.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const argv = require('minimist')(process.argv.slice(2));
+const log = require('fancy-log');
+const requestPromise = require('request-promise');
+const {cyan, green, yellow} = require('ansi-colors');
+const {gitCommitHash} = require('../../git');
+const {isTravisPullRequestBuild} = require('../../travis');
+
+const reportBaseUrl = 'https://amp-test-status-bot.appspot.com/v0/tests';
+
+const IS_INTEGRATION = !!argv.integration;
+const IS_LOCAL_CHANGES = !!argv['local-changes'];
+const IS_SAUCELABS = !!(argv.saucelabs || argv.saucelabs_lite);
+const IS_SINGLE_PASS = !!argv.single_pass;
+const IS_UNIT = !!argv.unit;
+
+function inferTestType() {
+  let type;
+  if (IS_UNIT) {
+    type = 'unit';
```

Comment:
@danielrozenberg Late realization: We don't explicitly use the `--unit` flag for `--local-changes` even though they are in fact unit tests. Therefore, we're not reporting status for `--local-changes`.

This can be fixed by adding the `--unit` flag here...

https://github.com/ampproject/amphtml/blob/66eb29d37ce623578d5b0953df80f61f682edb92/build-system/pr-check/local-tests.js#L69

... or by changing the logic of this function.